include/bluetooth/gatt.h

```diff
@@ -1132,6 +1135,9 @@ struct bt_gatt_indicate_params {
 	uint16_t len;
 	/** Private reference counter */
 	uint8_t _ref;
+#ifdef CONFIG_BT_EATT
+	enum bt_att_bearer_option bearer_option;
+#endif /* CONFIG_BT_EATT */
```

Comment:
I do wonder if we need this option for any other besides notifications. Indications, as well as other requests, will behave the exact same say regardless of whether it is an ATT or EATT bearer. 

The only difference will be the MTU, and the stack should be able to figure that out by itself.